packages/faustwp-cli/tests/healthCheck/validateFaustEnvVars.test.ts

```diff
@@ -45,8 +46,24 @@ describe('healthCheck/validateFaustEnvVars', () => {
 
     process.env.NEXT_PUBLIC_WORDPRESS_URL = 'http://headless.local';
 
-    validateFaustEnvVars();
+    await validateFaustEnvVars();
 
     expect(mockExit).toBeCalledTimes(0);
   });
+
+  it('logs an error when the secret key validation fails', async () => {
+
+    process.env.NEXT_PUBLIC_WORDPRESS_URL = 'https://headless.local';
+    process.env.FAUST_SECRET_KEY = 'invalid-secret-key';
+
+    fetchMock.post('https://headless.local/wp-json/faustwp/v1/validate_secret_key', {
+      status: 401,
+    });
+    
+    await validateFaustEnvVars();
+
+    // return expect(Promise.resolve(await validateFaustEnvVars())).toMatchInlineSnapshot(`healthCheck/validateFaustEnvVars logs an error when the secret key validation fails: Check to ensure your FAUST_SECRET_KEY matches your Faust Secret Key under wp-admin settings 1`);
+    return expect(Promise.resolve(validateFaustEnvVars())).toMatchSnapshot(`healthCheck/validateFaustEnvVars logs an error when the secret key validation fails: Check to ensure your FAUST_SECRET_KEY matches your Faust Secret Key under wp-admin settings 1`);
```

Comment:
How does this test work? Shouldn't be just checking if the program was exited or the error log was called instead of checking this snapshot?

Reply:
@theodesp, Here we mock the endpoint's response by providing a response status of 401- this gets passed to our function `validateFaustEnvVars`, and then we compare its response (which should be an error message) to the snapshot with the error message.

Please let me know if something is off-base, I'll be happy to tweak so we have good coverage. 👍

Reply:
Any further thoughts on the above item, @theodesp ?